Package.swift

```diff
@@ -106,7 +124,8 @@ let package = Package(
         .product(name: "ArgumentParser", package: "swift-argument-parser"),
         .product(name: "SwiftSyntax", package: "swift-syntax"),
         .product(name: "SwiftParser", package: "swift-syntax"),
-      ]
+      ],
+      linkerSettings: swiftformatLinkSettings
```

Comment:
Can this be made into a function that returns the desired value, so that its definition can be moved down to the bottom of the file? That way there's no noise before the package definition itself that average users don't care about.

Reply:
My goal was to keep the structure similar to that of [sourcekit-lsp’s Package.swift](https://github.com/apple/sourcekit-lsp/blob/main/Package.swift) and [swift-syntax’s Package.swift](https://github.com/apple/swift-syntax/blob/main/Package.swift) for consistency across the toolchain packages. 

I think moving those variables to be functions at the bottom of the file might be a good idea but I’d like to try that across all three packages first and see if it works for all of them. I’ll do it in a follow-up PR.